buildSrc/src/main/java/com/uber/okbuck/core/model/android/AndroidTarget.java

```diff
@@ -157,22 +157,24 @@ public AndroidTarget(Project project, String name) {
 
   @Override
   public Scope getMain() {
-    return Scope.from(
-        getProject(),
-        getBaseVariant().getRuntimeConfiguration(),
-        getSources(getBaseVariant()),
-        getJavaResources(getBaseVariant()),
-        getJavaCompilerOptions(getBaseVariant()));
+    return Scope.builder(getProject())
+        .configuration(getBaseVariant().getRuntimeConfiguration())
+        .sourceDirs(getSources(getBaseVariant()))
+        .javaResourceDirs(getJavaResources(getBaseVariant()))
+        .javaCompilerOptions(getJavaCompilerOptions(getBaseVariant()))
+        .build();
   }
 
   @Override
   public Scope getTest() {
-    return Scope.from(
-        getProject(),
-        getUnitTestVariant() != null ? getUnitTestVariant().getRuntimeConfiguration() : null,
-        getUnitTestVariant() != null ? getSources(getUnitTestVariant()) : ImmutableSet.of(),
-        getJavaResources(getUnitTestVariant()),
-        getJavaCompilerOptions(getUnitTestVariant()));
+    Scope.Builder builder = Scope.builder(getProject());
+    UnitTestVariant unitTestVariant = getUnitTestVariant();
+    if (unitTestVariant != null) {
+      builder.configuration(unitTestVariant.getRuntimeConfiguration());
+      builder.sourceDirs(getSources(getUnitTestVariant()));
+      builder.javaCompilerOptions(getJavaCompilerOptions(unitTestVariant));
+    }
```

Comment:
missing `resources`

Reply:
good catch. mind fixing in a follow up?

Reply:
Fixed in https://github.com/uber/okbuck/commit/1d25f8918783a3870d46709159f6101fc94fbf8a

Reply:
for null, we return empty map which we dont need to handle anymore as its the default. Will clean up that code later